models/engine/db_storage.py

```diff
@@ -74,3 +74,14 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """object based on the class name and its ID, or None"""
```

Comment:
- You should try to follow the Google Docstring convention as indicated in previous projects

> (`https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html`)